src/payment/models/bank-tx/dto/update-bank-tx.dto.ts

```diff
@@ -10,4 +12,15 @@ export class UpdateBankTxDto {
   @IsInt()
   @ValidateIf((p) => p.type === BankTxType.BUY_CRYPTO)
   buyId: number;
+
+  @IsNotEmptyObject()
+  @ValidateIf((p) => p.type === BankTxType.BANK_TX_RETURN)
+  @ValidateNested()
+  @Type(() => EntityDto)
+  chargebackBankTx: BankTx;
```

Comment:
Should we just use the chargebackBankTxId here (same as buyId in case of BuyCrypto)?